src/main/scala/firrtl/transforms/DeadCodeElimination.scala

```diff
@@ -53,7 +53,7 @@ class DeadCodeElimination extends Transform
          Dependency(passes.VerilogPrep),
          Dependency[firrtl.AddDescriptionNodes] )
 
-  override def invalidates(a: Transform) = false
+  override def invalidates(a: Transform) = a == ResolveKinds
```

Comment:
Do you know if this had anything todo with the use of `ExprKind` instead of `MemKind` on line 134?